packages/genesys-spark-tokens/data/ui/data_display.json

```diff
@@ -993,12 +993,6 @@
       }
     }
   },
-  "global_nav": {
-    "side_menu": {
-      "value": "linear-gradient(135deg, {color.emerald.500} 0%, {color.emerald.200} 100%)",
-      "type": "color"
-    }
-  },
```

Comment:
@VirtualClover this upgrade found an issue in the data files. This entry was silently stopping all the `global_nav.side_menu.XXXXX` tokens in `navigation.json`
I moved it into `navigation.json` and called it `global_nav.side_menu.background_color`